fixtures/algam/slim-par-1210-rgbwau.json

```
@@ -0,0 +1,110 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Slim PAR 1210 RGBWAU",
+  "shortName": "SLIMPAR-1210-HEX ",
+  "categories": ["Color Changer", "Dimmer", "Strobe"],
+  "meta": {
+    "authors": ["Francois Bouffard"],
+    "createDate": "2021-02-01",
+    "lastModifyDate": "2021-02-01"
+  },
+  "links": {
+    "manual": [
+      "https://www.algam-webstore.fr/algam-lighting-projecteur-a-led-12x10w-6-en-1-rgbwau-slim-552310.html"
+    ],
+    "productPage": [
+      "https://www.algam-webstore.fr/algam-lighting-projecteur-a-led-12x10w-6-en-1-rgbwau-slim-552310.html"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=D0Z_f59QR9w"
+    ]
+  },
+  "physical": {
+    "dimensions": [227, 220, 70],
+    "weight": 1.5,
+    "power": 48,
+    "DMXconnector": "3-pin"
+  },
+  "availableChannels": {
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Amber": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Amber"
+      }
+    },
+    "UV": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "UV"
+      }
+    },
+    "Strobe": {
+      "capability": {
+        "type": "StrobeSpeed",
+        "speedStart": "4Hz",
+        "speedEnd": "500Hz"
+      }
```

```suggestion
      "capabilities": [
        {
          "dmxRange": [0, 0],
          "type": "ShutterStrobe",
          "shutterEffect": "Open"
        },
        {
          "dmxRange": [1, 255],
          "type": "ShutterStrobe",
          "shutterEffect": "Strobe",
          "speedStart": "slow",
          "speedEnd": "fast"
        }
      ]
```

I don't see anywhere the manual or specs say the range of strobe speeds, and 500Hz seems awfully high to me.